installers/binstall/scripts/nix/install.sh

```diff
@@ -63,13 +63,15 @@ download_binary_and_run_installer() {
     local _dir="$(mktemp -d 2>/dev/null || ensure mktemp -d -t rover)"
     local _file="$_dir/input.tar.gz"
     local _rover="$_dir/rover$_ext"
+    local _safe_url
 
-    say "downloading rover from $_url" 1>&2
+    _safe_url=$(echo "$_url" | sed  -E 's|https://[^@]+@|https://|')
```

Comment:
I'm commenting on this PR even though it's the same as `router`.  I'm a uncomfortable adding `sed` as a dependency here, as the absence of `sed` will make entire install script fail and it is a net-new dependency to this relatively critical script (which can be a breaking change).  At the very least, it should be added to the `need_cmd` set of required commands at the top, but also, this command has known differences between macOS and Linux.

Is it possible to accomplish the same thing using `awk`, which is already an existing requirement:

https://github.com/apollographql/rover/blob/dd5a172b03a7ac4e684c3ef09646c096035e5dea/installers/binstall/scripts/nix/install.sh#L35

Reply:
Alternatively, since it will be obscuring information that might actually useful, maybe an another approach is just to say, `The APOLLO_ROVER_DOWNLOAD_GITHUB_HOST environment variable was set.  See value for full details.`?